presto-hive/src/main/java/io/prestosql/plugin/hive/ViewReaderUtil.java

```diff
@@ -113,6 +125,16 @@ public ConnectorViewDefinition decodeViewData(String viewData, Table table, Cata
             byte[] bytes = Base64.getDecoder().decode(viewData);
             return VIEW_CODEC.fromJson(bytes);
         }
+
+        public static ConnectorMaterializedViewDefinition decodeMaterializedViewData(String data)
```

Comment:
This method doesn't belong in this commit.

Reply:
Moving to iceberg connector commit